cdap-ui/app/cdap/components/Description/index.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {Component, PropTypes} from 'react';
+require('./Description.scss');
+import T from 'i18n-react';
+
+export default class Description extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      description: this.props.description
+    };
+  }
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.description !== this.state.description) {
+      this.setState({
+        description: nextProps.description
```

Comment:
Talked to @elfenheart We will remove this when we add loading animation on entity switch